src/Components/Articles.jsx

```diff
@@ -0,0 +1,31 @@
+import Thumbnail from "./Thumbnail";
+import getArticles from "../../api";
+import { useEffect, useState } from "react";
+
+const Articles = () => {
+    const [articlePage, setArticlePage] = useState([])
+
+    useEffect(()=>{
+        getArticles().then(({articles}) => {
+            setArticlePage(articles)
+        })
+    }, [])
+    console.log(articlePage)
```

Comment:
Could throw a loading state in here, users liked to see when background processes are still ongoing